packages/salesforcedx-vscode-core/src/commands/functions/functionService.ts

```diff
@@ -0,0 +1,47 @@
+/*
+ * Copyright (c) 2020, salesforce.com, inc.
+ * All rights reserved.
+ * Licensed under the BSD 3-Clause license.
+ * For full license text, see LICENSE.txt file in the repo root or https://opensource.org/licenses/BSD-3-Clause
+ */
+
+/**
+ * A running task that can be terminated
+ */
+interface Terminable {
+  terminate: () => Promise<void>;
+}
+
+export class FunctionService {
```

Comment:
Can we have a test for this class?

Reply:
FunctionService is just an implementation detail for stopping the server, and is suspect to change when we employed VS Code tasks later. All methods here are already covered. (the diff coverage is 95.23%)